### DISCUSSION TITLE
Welcome to Anstagram Discussions!

### DISCUSSION THREAD

**Discussion by shindesharad71** [shindesharad71/Anstagram#47] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Bikki-coder-dev** [comment]
for the frontend and backend why its not running even after installing the required packages i.e npm install -g @nrwl/cli ?
